httpbin/core.py

```diff
@@ -290,10 +290,10 @@ def view_headers():
       - application/json
     responses:
       200:
-        description: The Rrquest's IP Address.
+        description: The Request's IP Address.
```

Comment:
Good catch on the typo. This looks like it's actually a bad description too. We want this to be "The request's headers."